src/Datasource/InvalidPropertyInterface.php

```diff
@@ -17,6 +17,9 @@
 /**
  * Describes the methods that any class representing a data storage should
  * comply with.
+ *
+ * @method \Cake\Datasource\EntityTrait getInvalid()
```

Comment:
Do you have some docs for me on how to create this annotations correctly? I never used an IDE so I have no clue what is needed here.